m4_sessions/1705-inning/group_eight.md

```diff
@@ -1,22 +1,25 @@
 # Group Eight
 
 ## List your group members here:
-Andrew Wooten, Kenia Farias, Pamela Lovett, Casey Macaulay 
+Andrew Wooten, Kenia Farias, Pamela Lovett, Casey Macaulay
 
 ## Topic
 Danger of a Single Story
 
 ## Session Outcomes
 - Identify the single stories/stereotypes in the tech industry
 - (Activity) Find facts and narratives that counter the single story
-- Encourage students to get to know eachother as people 
+- Encourage students to get to know eachother as people
 
 ## Session Outline
 General Outline:
-- State objective for the session
-- Show video (not the whole thing) [TED](https://www.ted.com/talks/chimamanda_adichie_the_danger_of_a_single_story)
-- Potentially do this activity/show this clip (Second Clip)[https://www.youtube.com/watch?v=hD5f8GuNuGQ]
-- Small group discussions w/ questions (3-4 people, count off)
+- Intro (5 min)
+- Show video (20 min) [TED](https://www.ted.com/talks/chimamanda_adichie_the_danger_of_a_single_story)
+- Writing time (5 min)
+- Small Groups (10 min)
+- Line Exercise (10 min)
+- Final Question Closing (10 min)
+
 
 
 Potential Discussion questions:
```

Comment:
I know you are working hard on this and I'm sure it will be great.  Just go back in and add the questions so you don't have to think about the flow of the conversation.  Also good to think about how groups will be sitting and if you want them in a specific physical space.  You could do the session in Clarke if you want to have more room for the line activity.  Let me know!